ecl2df/compdat.py

```diff
@@ -745,10 +745,10 @@ def applywelopen(
       /
 
     This deck would define two wells where OP1 and OP2 have two connected grid cells
-    each. Although the COMPDAT defines all connections to be open, WELOPEN overwrites
-    this for OP2 since items 3-5 are not defaulted. The connections in OP1 are untouched
-    when items 3-7 are defaulted in WELOPEN, then the statement applies only to the well
-    state itself, which is not covered by the compdat dataframe.
+    each. The first welopen statment acts on the whole well, closing both the well and
+    the connections. If this statement used STOP instead of SHUT, the connections would
+    be left open. The second welopen statement acts on a single connection. Here SHUT
+    and STOP would give the same result. The Eclipse manual is unclear about this logic.
```

Comment:
State that the behaviour in this class is proven to be correct in the simulator`?